### DISCUSSION TITLE
How can I simulate a large molecular drug?

### DISCUSSION THREAD

**Discussion by csungeun** [Open-Systems-Pharmacology/Forum#1566] [How-Tos] [answered]
hi, I want to build a PBPK model of a large molecular drug.
First of all, I am practicing based on the thesis(Please refer to the attached file), but it is not going well.
Help me....

[problem_and_thesis.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/12022386/problem_and_thesis.zip)

**Comment by prvmalik** [comment, marked as answer]
That's right, this model only works in version 9.1 or lower.

Also I'd say it's one of the more complicated large molecule models available.

In this particular example, you have to pick the "Model for Large Molecules" on the slide of the "Configure Simulation" pane.

**Reply by csungeun** [reply]
Thank you!!!!!!!
I hope everything would be a success.

**Comment by duoduo982** [comment]
Hello. How's it going?
